packs/st2/actions/lib/action.py

```diff
@@ -20,21 +20,24 @@ def __init__(self, config):
         self.client = self._get_client()
 
     def _get_client(self):
-        base_url, api_url = self._get_st2_urls()
+        base_url, api_url, auth_url = self._get_st2_urls()
         token = self._get_auth_token()
-        return self._client(base_url=base_url, api_url=api_url, token=token)
+        return self._client(base_url=base_url, api_url=api_url,
+                            auth_url=auth_url, token=token)
 
     def _get_st2_urls(self):
         # First try to use base_url from config.
         base_url = self.config.get('base_url', None)
```

Comment:
Also, I'm a bit confused.

Did you intentionally remove `base_url` from the config? Because we still retrieve it from the config here so it looks like we still support that option.